app/src/main/java/org/listenbrainz/android/ui/navigation/AppNavigation.kt

```diff
@@ -37,5 +38,8 @@ fun AppNavigation(
         composable(route = AppNavigationItem.Profile.route){
             ProfileScreen(shouldScrollToTop = shouldScrollToTop)
         }
+        composable(route = AppNavigationItem.Feed.route){
```

Comment:
Replace the Home screen with the Feed screen. I'll move the content from there to Explore screen

Reply:
Alright